boat-maven-plugin/pom.xml

```diff
@@ -155,8 +155,8 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <source>${maven-compiler-plugin.source}</source>
```

Comment:
The maven-compiler plugin supports properties `maven.compiler.source` and  `maven.compiler.target` https://maven.apache.org/plugins/maven-compiler-plugin/compile-mojo.html